components/admin_console/admin_sidebar/admin_sidebar.tsx

```diff
@@ -145,12 +161,12 @@ class AdminSidebar extends React.PureComponent {
             currentSiteName = ' - ' + this.props.siteName;
         }
 
-        document.title = Utils.localizeMessage('sidebar_right_menu.console', 'System Console') + currentSiteName;
+        document.title = localizeMessage('sidebar_right_menu.console', 'System Console') + currentSiteName;
     }
 
     visibleSections = () => {
         const {config, license, buildEnterpriseReady, consoleAccess, adminDefinition, cloud} = this.props;
-        const isVisible = (item) => {
+        const isVisible = (item: any) => {
```

Comment:
Reminder: In the original PR, we decided to keep as any until AdminDefinition type is created, Please see the discussion [here](https://github.com/mattermost/mattermost-webapp/pull/10162/files#r850695150)